temporal-testing/build.gradle

```diff
@@ -0,0 +1,187 @@
+// Run 'gradle checkUpdates' to find out which dependencies have newer versions
+plugins {
+    id 'java-library'
+    id 'net.ltgt.errorprone' version '1.3.0'
+    id 'net.minecrell.licenser' version '0.4.1'
+    id 'com.palantir.git-version' version '0.12.3'
+    id 'maven-publish'
+    id 'signing'
+    id 'de.marcphilipp.nexus-publish' version '0.4.0'
+    id 'name.remal.check-updates' version '1.1.6'
+}
+
+apply plugin: 'maven-publish'
+apply plugin: 'de.marcphilipp.nexus-publish'
+apply plugin: 'java'
+
+if (hasProperty('signing.keyId')) {
+    apply plugin: 'signing'
+    signing {
+        sign configurations.archives
+    }
+}
+
+group = 'io.temporal'
+version = getVersionName()
+archivesBaseName = "temporal-testing"
+
+description = '''Temporal Workflow Java SDK testing'''
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+    withJavadocJar()
+    withSourcesJar()
+}
+
+dependencies {
+    errorproneJavac('com.google.errorprone:javac:9+181-r4173-1')
+    errorprone('com.google.errorprone:error_prone_core:2.4.0')
+
+    api project(':temporal-sdk')
```

Comment:
I believe most of the other dependencies are already part of temporal-sdk. Can we remove them from here?

Reply:
We need errorprone here as we are using it directly in that build file. We can probably configure it in the root project instead, but that would take some time.